docs/index.md

```diff
@@ -0,0 +1,151 @@
+---
+hide_title: true
+sidebar_position: 0
+title: Overview
+---
+
+# Documentation
```

Comment:
Why not put this text in the title and remove the hide_title?

Also, I suggest something more specific, like Macrometa Documentation as the overview.

Reply:
@diana-macrometa thanks, "Macrometa" and "Docs" are already safely established by the site navigation scheme. This is an overview page, but since it's also the base page of the docs site it should mirror that placement with a clear page title. I've also set it up this way to avoid name overload, which is a bit unsightly:

<img width="844" alt="image" src="https://user-images.githubusercontent.com/180438/219747654-18cddd4a-db0c-419e-b5c1-01c3fa582dd6.png">